controller/controller.js

```diff
@@ -38,4 +38,23 @@ const getArticles = (req, res, next) => {
     .catch(next);
 };
 
-module.exports = {getTopics,getApis,getArticles,getArticleById}
+const patchArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (inc_votes === undefined) {
+    return next({ status: 400, msg: 'Bad Request: Missing inc_votes in request body' });
+  }
+
+  if (typeof inc_votes !== 'number') {
+    return next({ status: 400, msg: 'Bad Request: Invalid data type for inc_votes' });
+  }
```

Comment:
We want to invoke next with the error here to be handled by the next error handler rather than returning it.